### PR TITLE
Show Album and Artist info in the menu

### PR DESCRIPTION
**Pull request by bake**

### CODE REVIEW

**Comment by cheeaun** [comment]
Wait, this works for you with latest version of Electron? 😱

**Comment by bake** [comment]
It works with Electron 0.33.1 :)

File: app.js [hunk 1]
```diff
@@ -53,12 +55,23 @@ function truncateName(name, charsLimit){
...
+
 var currentName = '', currentState = 'paused';
 itunes.on('playing', function(data){
   currentState = 'playing';
   currentName = data.name;
   appTray.setTitle('▶ ' + truncateName(currentName) + '  ');
 
+  // Rebuild the whole context-menu on every change. Two times.
+  updateMenuLabel(0, (data.album) ? 'Album: ' + data.album : '');
+  updateMenuLabel(1, (data.artist) ? 'Artist: ' + data.artist : '');
```
Comment (cheeaun) [inline app.js]: Would be great if you could remove all my commented code below too :smile:

**Comment by bake** [comment]
Done

File: app.js [hunk 2]
```diff
@@ -53,28 +55,23 @@ function truncateName(name, charsLimit){
...
-  // } else {
-  //   albumMenu.visible = false;
-  // }
-  // if (data.artist){
-  //   artistMenu.label = 'Artist: ' + data.artist;
-  //   artistMenu.visible = true;
-  // } else {
-  //   artistMenu.visible = false;
-  // }
+  // Rebuild the whole context-menu on every change. Two times.
```
Comment (cheeaun) [inline app.js]: Hmm, would it be better if it's only rebuilt once?

**Comment by bake** [comment]
Like this? We could also remove [#64](https://github.com/BakeRolls/kyoku/blob/6c5d0ce192b33f281c58b185ea96f936671fdf8a/app.js#L64) and just use visibility in #78, #79, #87 and #88.

File: app.js [hunk 3]
```diff
@@ -53,32 +55,40 @@ function truncateName(name, charsLimit){
...
+    { index: 1, template: { label: (data.artist) ? 'Artist: ' + data.artist : '' } }
+  ]);
 });
+
 itunes.on('paused', function(data){
   currentState = 'paused';
   appTray.setTitle(defaultTitle);
+  updateMenu([
+    { index: 0, template: { label: '' } },
+    { index: 1, template: { label: '' } }
```
Comment (cheeaun) [inline app.js]: Perhaps can set `visible: false` instead of `label: ''`, which I think is more explicit?

**Pull request merged**